standstill_action.talon-list

```diff
@@ -0,0 +1,28 @@
+list: user.clickless_mouse_action
+-
+# Actions relevant for the single stage clicker only
+
+left: left_click
+left click: left_click
+
+right: right_click
+right click: right_click
+
+hover: hover
+
+duke: left_dubclick
```

Comment:
Some of these might conflict with default community commands. I'm not sure how or if we should we worry about it. Just noting for when I get around to testing